target/linux/brcm47xx/image/Makefile

```diff
@@ -702,15 +778,19 @@ endef
 TARGET_DEVICES += linksys-wrt320n-v1
 
 define Device/linksys-e900-v1
-  DEVICE_TITLE := Linksys E900 v1
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := E900
+  DEVICE_VARIANT := v1
   $(Device/linksys)
   DEVICE_ID := E900
   VERSION := 1.0.4
 endef
 TARGET_DEVICES += linksys-e900-v1
 
 define Device/linksys-e1000
-  DEVICE_TITLE := Linksys E1000 v1 v2 v2.1
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := E1000
+  DEVICE_VARIANT := v1 v2 v2.1
```

Comment:
I would prefer slashes here: v1/v2/v2.1
Despite, I would remove the v2.1 here, as subversions are normally included anyway.